usr/lib/pkcs11/cca_stdll/cca_specific.c

```diff
@@ -2337,7 +2337,7 @@ CK_RV ccatok_hmac(STDLL_TokData_t * tokdata, SIGN_VERIFY_CONTEXT * ctx,
     }
 
     if (sign && !sig_len) {
-        TRACE_ERROR("%s received bad argument(s)\n", __FUNCTION__);
+        TRACE_ERROR("%s received bad argument(s)\n", __func__);
```

Comment:
isn't the function name anyway part of each trace statement ?

Reply:
yes it is. i wont modify the trace output with this PR.